dvc/info.py

```diff
@@ -32,6 +32,7 @@ def get_dvc_info():
         f"Platform: Python {platform.python_version()} on {platform.platform()}",
         f"Subprojects:{_get_subprojects()}",
         f"Supports:{_get_supported_remotes()}",
+        f"Config:{_get_config_dirs()}",
```

Comment:
this format is completely crazy, need to switch to dict and dump as json/yaml instead in the future.